examples/quickstart/jupyter-notebooks/notebooks/03-query/03-dataDistribution.ipynb

```diff
@@ -0,0 +1,606 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea9192f0-8a5f-4954-b63c-133964cc4b28",
+   "metadata": {},
+   "source": [
+    "### Estimate multiple rank positions\n",
+    "\n",
+    "In order to estimate multiple rank positions, the [`DS_CDF`](https://druid.apache.org/docs/latest/querying/sql-functions#ds_cdf) function is available.\n",
+    "\n",
+    "Run the cell below to use this function to return an `ARRAY` of ranking positions for 6 different delays per `Reporting_Airline`, from 30 minutes through to 16 hours. The results show what percentage of reported delays in the dataset were under each of the lengths of time we have specified.\n",
+    "\n",
+    "The `MAX` is added as a sorting criteria."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95127870-05f0-434a-927a-044205b7104b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT\n",
+    "  Reporting_Airline,\n",
+    "  DS_CDF(DS_QUANTILES_SKETCH(Delay), 30, 60, 120, 240, 480, 960) AS \"cdf\",\n",
+    "  MAX(Delay) AS \"Longest Delay\"\n",
+    "FROM \"example-flights-quantiles\"\n",
+    "WHERE TIME_FLOOR(\"__time\", 'P1W') = TIMESTAMP '2005-11-07'\n",
+    "AND Delay > 0\n",
+    "GROUP BY 1\n",
+    "ORDER BY 3 DESC\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ecc3b60a-54f8-4e91-9618-8afc635a7b02",
+   "metadata": {},
+   "source": [
+    "## Estimating histograms\n",
+    "\n",
+    "When wanting to understand the shape of the distribution, Druid provides the [`DS_HISTOGRAM`](https://druid.apache.org/docs/latest/querying/sql-functions#ds_histogram) function, which also acts on an underlying Quantiles sketch.\n",
+    "\n",
+    "Run the following cell to see how the `DS_HISTOGRAM` function estimates the buckets of `Distance` of flights taking off from Los Angeles."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "efaca521-f5c3-4d61-8fb6-f5ed77b1760c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT Origin,\n",
+    "  DS_HISTOGRAM(DS_QUANTILES_SKETCH(Distance), 250, 500, 750, 1000, 1250, 1500, 1750, 2000) AS Histogram\n",
+    "FROM \"example-flights-quantiles\"\n",
+    "WHERE \"Origin\" = 'LAX'\n",
+    "AND TIME_FLOOR(\"__time\", 'P1W') = TIMESTAMP '2005-10-31'\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea9b2ef0-b323-4b1c-b8dc-755e6fff32d7",
+   "metadata": {},
+   "source": [
+    "The results show an estimated count of the number of flights in each of the `Distance` buckets specified, plus one final bucket for the remainder.\n",
+    "\n",
+    "Combining `DS_QUANTILES_SKETCH` with `FILTER` focuses the distribution being analysed.\n",
+    "\n",
+    "Run the next cell which produces histograms for two specific `Reporting_Airline` with flights from Los Angeles."
```

Comment:
For this cell and the previous one, could be cool to see them in histogram form: https://matplotlib.org/stable/gallery/statistics/hist.html#sphx-glr-gallery-statistics-hist-py